CHANGELOG.md

```diff
@@ -5,6 +5,145 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 8.0.0-alpha.10
+
+_Jan 30, 2025_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 📚 Documentation improvements
```

Comment:
```suggestion
- 🎨 Data Grid theming improvements and default background color
- 📚 Documentation improvements
```

Reply:
I don't know if we need the default background color bit in the highlight, happy to exclude that part.